src/lib/utils/string_functions.hpp

```diff
@@ -0,0 +1,18 @@
+#include <boost/algorithm/string/split.hpp>
+#include <boost/algorithm/string/trim.hpp>
+
+#include <string>
+
+namespace opossum {
+
+std::vector<std::string> trim_and_split(const std::string& input) {
```

Comment:
How does this deal with multiple spaces within the input? Also, implement in cpp?

Reply:
Fixed and tested.